keadm/cmd/keadm/app/cmd/edge/join.go

```diff
@@ -403,3 +432,15 @@ func runEdgeCore() error {
 	klog.Infoln(tip)
 	return nil
 }
+
+func createBootstrapFile(opt *common.JoinOptions) error {
+	bootstrapFile := constants.BootstrapFile
+	_, err := os.Create(bootstrapFile)
+	if err != nil {
+		return err
+	}
+
+	// write token to bootstrap-edgecore.conf file
+	token := []byte(opt.Token)
+	return os.WriteFile(bootstrapFile, token, 0666)
```

Comment:
`0640` is the proper permission for the file.

Reply:
fixed